web/src/typing.ts

```diff
@@ -81,7 +81,10 @@ function send_stream_typing_notification(
     topic: string,
     operation: "start" | "stop",
 ): void {
-    const stream = stream_data.get_sub_by_id(stream_id)!;
+    const stream = stream_data.get_sub_by_id(stream_id);
+    if (stream === undefined) {
+        return;
+    }
```

Comment:
What's the circumstance when this would fire? I assume it's for if we lost access to the channel and are failing to send a stopped notification? Deserves a comment.